### PR TITLE
Project - Carolina Cornejo Castellano

### PR DESCRIPTION
**Pull request by castellco**
This is my preliminary report. It includes replication, enhancements, alternative(s) and discussion. I remain attentive to any correction or improvement proposal.

### CODE REVIEW

**Comment by Enchufa2** [comment]
Very nice work! I've moved the stuff to the 2022 folder and made some adjustments to avoid the weird `path/to/folder` path for the two images. So please **click Pull** in RStudio's git panel to get this commit from GitHub, and wait for further comments here.

File: _projects/2022/100481925/100481925.Rmd [hunk 1]
```diff
@@ -0,0 +1,1481 @@
...
+As for the article on the _Visual Capitalist_ website, it has as newsworthiness -in Spanish journalist jargon we call it "la pepa"- to show the distribution of Americans' spending in 2021. Although the title of the article does not specify that the information is from the previous year -it was written and published in 2022, but the information is from 2021- this type of generalization or omissions are common in the titles and/or in the headlines of journalistic articles: they are licenses that journalists give themselves to attract the reader's attention. The content of the article clearly specifies the source of information—the PDF—and that it refers to 2021. So it's not a time series, it's just a picture of that moment.
+
+<div>
+  <center><img src="https://github.com/carolinacornejocastellano/bump-chart/blob/main/original.png?raw=true" width="100%">
+    <figcaption>
+    <span class="caption">Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.</span>
+    <i class="photo-credit">Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/</i>
+  </figcaption>
+  </p>
+</div>
```
Comment (Enchufa2) [inline _projects/2022/100481925/100481925.Rmd]: I would prefer if you let pandoc generate the HTML for these figures. So please use Markdown syntax for all the images you add with divs. For instance, this one would be just:

```markdown
![Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.  Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/](https://github.com/carolinacornejocastellano/bump-chart/blob/main/original.png?raw=true){.external width="100%"}
```

Note the `.external` tag. This adds a special border intended for external images. So do not add it to your own images.

Also, I would prefer if posts do not depend on external resources as much as possible. So please download all these images and place them into the `_projects/2022/100481925/images` folder I've created. Then you can reference them with a path relative to the Rmd, so the code above would be just

```markdown
![Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.  Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/](images/original.png){.external width="100%"}
```

File: _projects/2022/100481925/100481925.Rmd [hunk 2]
```diff
@@ -0,0 +1,1481 @@
...
+library(ggplot2) # for data graphics, based on the Grammar of Graphics (Wilkinson 2005).
+library(ggtext) # provides markdown and HTML text rendering
+library(grid) # for adding the two white lines at the top and bottom
+```
+
+Then, the data was imported. As mentioned above, the data is only a pdf, so the values had to be transcribed manually. 
+
+```{r}
+df <- read_csv(
+  "https://github.com/carolinacornejocastellano/bump-chart/raw/main/data.csv")
```
Comment (Enchufa2) [inline _projects/2022/100481925/100481925.Rmd]: The same for the data, meaning: please download the dataset and put it here with the rest of the files, then reference that with a path relative to the Rmd file.

File: _projects/2022/100481925/100481925.Rmd [hunk 3]
```diff
@@ -0,0 +1,1481 @@
...
+  "<span style='font-size: 9.55pt'>**Generation X**</span><br>1965 to 1980<br>**$83,357**",
+
+  "<span style='font-size: 9.55pt'>**Millennials**</span><br>1981 to 1996<br>**$69,061**",
+
+  "<span style='font-size: 9.55pt'>**Generation Z**</span><br>1997 or later<br>**$41,636**"
+)
+```
+
+```{r}
+x_lab <- c("<span style='font-size: 10pt'>**Generation**</span><br>Birth Year Range<br>**Average Annual Expenditure**")
```
Comment (Enchufa2) [inline _projects/2022/100481925/100481925.Rmd]: Note that there are some lines with strings that are too wide. You can wrap them by pasting several shorter strings together using `paste()` or `paste0()`.

Please check the rest of the post for other cases, specifically very long titles and subtitles.

File: _projects/2022/100481925/100481925.Rmd [hunk 4]
```diff
@@ -0,0 +1,1481 @@
...
+  </figcaption>
+  </p>
+</div>
+
+However, there is another important piece of information in Preethi Lodha's graph, which is the dollar amount spent by each generation for each expenditure category. This is shown within each point and is undoubtedly important if one wants to know how spending is distributed; however, that oscillation is not represented and can be misleading. This point will be discussed later.
+
+## What can be seen here? Encodings, channels and strengths
+There are 14 spending categories represented on the Y axis and 5 generations or age ranges showed on the X axis, above the graph and below the title and subtitle, from the Silent generation (the oldest) to the Z generation.
+
+According to Cleveland (1985), graphs encode data in distinct elements, such as symbols, colors, position, etc. When we study a graphic we perform several mental-visual tasks to extract quantitative information. Some of the basic judgements we perform to decode quantitative information are 1) angle, 2) area, 3) color hue, 4) color saturation, 5) density, 6) length, 7) position along a common scale, 8) position along identical, nonaligned scales, 9) slope, 10) volume (Cleveland, 1985).
```
Comment (Enchufa2) [inline _projects/2022/100481925/100481925.Rmd]: There is a special syntax to add citations to R Markdown documents that not only generates links to the references, but also generates the reference list automatically at the end. See [the documentation](https://rstudio.github.io/distill/citations.html) please. It's a bit more of work but it's worth it.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Please **click Pull** first off to get my commit, then address the comments below, knit the document again and commit and push the changes.

**Comment by castellco** [comment]
Hi, Iñaki. I have fixed the Rmd according to your feedback: the citations, the code width and the paths of the external images, which were copied to the _images_ folder. The csv of the data and the CSL file of the citations were added to the same folder as the Rmd. Is it right? Thanks in advanced for your comments.

**Review by Enchufa2 — APPROVED** [review]
Splendid work, congrats!

**Comment by Enchufa2** [comment]
And here you go: https://csslab.uc3m.es/dataviz/projects/2022/100481925/

**Pull request merged**